### ISSUE TITLE
Crash on startup, may be bounty-related?

### ISSUE THREAD

**Issue by Atashkada** [RkShaRkz/Starsector-IntelliJ-Template#18]
Mod version: 3.3.7

No savegame, it crashed on new game startup. Attaching log and modlist.

[starsectorlog.zip](https://github.com/user-attachments/files/16119882/starsectorlog.zip)
[enabled_mods.json](https://github.com/user-attachments/files/16119883/enabled_mods.json)

I'm using mikohime java23 and changed some of the vayra configs, the json checker website said it was still a valid json after. Also attaching it.

[VAYRA_SETTINGS.zip](https://github.com/user-attachments/files/16119978/VAYRA_SETTINGS.zip)

Repro steps are trying to start a game. I somewhat doubt the options I picked influenced it going by the log, but it was Kesteven Start, level 6 preset, RAT comradery background.

From what I can tell, a stacktrace is simply the log uploaded above, if it is something please contact me.

**Comment by RkShaRkz** [comment]
well, a stacktrace looks just like this:
```
4998314 [Thread-2] ERROR com.fs.starfarer.combat.CombatMain  - java.util.ConcurrentModificationException
java.util.ConcurrentModificationException: null
        at java.base/java.util.ArrayList$Itr.checkForComodification(ArrayList.java:1096) ~[?:?]
        at java.base/java.util.ArrayList$Itr.next(ArrayList.java:1050) ~[?:?]
        at data.scripts.campaign.intel.VayraPersonBountyManager.checkForFuckedUpParticipants(VayraPersonBountyManager.java:76) ~[?:?]
        at data.scripts.campaign.intel.VayraPersonBountyManager.advance(VayraPersonBountyManager.java:268) ~[?:?]
        at data.scripts.VayraMergedModPlugin.onNewGameAfterTimePass(VayraMergedModPlugin.java:343) ~[?:?]
        at com.fs.starfarer.campaign.save.CampaignGameManager.o00000(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.title.TitleScreenState.dialogDismissed(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.O.dismiss(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.impl.for.dismiss(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.campaign.save.if.actionPerformed(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.o0O0.buttonPressed(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.int.├Æ00000(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.int.processInput(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.ui.W.o00000(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.BaseGameState.traverse(Unknown Source) ~[port_obf.jar:?]
        at com.fs.state.AppDriver.begin(Unknown Source) ~[port.common_obf.jar:?]
        at com.fs.starfarer.combat.CombatMain.main(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.StarfarerLauncher.super(Unknown Source) ~[port_obf.jar:?]
        at com.fs.starfarer.StarfarerLauncher$1.run(Unknown Source) ~[port_obf.jar:?]
        at java.base/java.lang.Thread.run(Thread.java:1575) [?:?]
 ```
        
the whole log may contain much more (useless) information but alright. cc @Atashkada

**Comment by RkShaRkz** [comment]
#19 and #20 are copies of this issue. fixed, just not commited yet. If you're keen on continuing with your game, ping me on discord for a new JAR.

**Comment by RkShaRkz** [comment]
taken care of by PR https://github.com/RkShaRkz/Starsector-IntelliJ-Template/pull/21 (commit 47a1ca5)

Will close when PR gets merged in. Will hold off releasing a new version since i want to take a good jab at the vanishing planet conditions too. Ping me on discord for hotfix if you're blocked, it's been confirmed as resolving the issue.

**Comment by RkShaRkz** [comment]
[jars.zip](https://github.com/user-attachments/files/16229804/jars.zip)
hotfix here.

**Comment by RkShaRkz** [comment]
Issue was resolved by 8bc9dbe - closing until it resurfaces on new 3.3.8 version.
